src/test/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloudFIPSTest.java

```diff
@@ -0,0 +1,109 @@
+package org.csanchez.jenkins.plugins.kubernetes;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.assertThrows;
+
+import hudson.ExtensionList;
+import io.jenkins.cli.shaded.org.apache.commons.io.FileUtils;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Paths;
+import jenkins.security.FIPS140;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.FlagRule;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.LocalData;
+
+public class KubernetesCloudFIPSTest {
+
+    @ClassRule
+    public static FlagRule<String> fipsFlag = FlagRule.systemProperty(FIPS140.class.getName() + ".COMPLIANCE", "true");
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    @Test
+    @Issue("BEE-73460")
```

Comment:
**note:** Irrelevant in OSS, use `JENKINS-` reference instead here.

Reply:
ups
```suggestion
    @Issue("JENKINS-73460")
```